client/src/components/Page/index.tsx

```diff
@@ -1,30 +1,27 @@
 import React from 'react';
 import { withStyles, createStyles } from '@material-ui/core/styles';
-import /*type*/ { WithStyles, Theme } from '@material-ui/core/styles';
+import { /*type*/ WithStyles, Theme } from '@material-ui/core/styles';
```

Comment:
Unfortunately, prettier doesn't understand this so it tosses the comment in the brackets. I'm thinking it shouldn't be a big problem since we can still search and replace once the feature is available.